packages/app/cypress/e2e/navigation.cy.ts

```diff
@@ -1,20 +0,0 @@
-import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
```

Comment:
This file has been redundant for a while with the addition of tests in top-nav.cy.ts.